### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by dyladan**
:robot: I have created a release \*beep\* \*boop\*
---
<details><summary>@opentelemetry/contrib-test-utils: 0.26.0</summary>

### Features

* upstream mocha instrumentation testing plugin from ext-js ([@blumamir](https://www.github.com/blumamir)) [#621](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/621) ([#670](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/670)) ([7edf984](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/7edf98425d77aaa1d74a959195e87b6079daabcd))
* upstream mocha instrumentation testing plugin from ext-js [#621](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/621) ([#669](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/669)) ([a5170c4](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/a5170c494706a2bec3ba51e59966d0ca8a41d00e))
</details>
<details><summary>@opentelemetry/instrumentation-aws-lambda: 0.26.0</summary>

### Features

* upstream mocha instrumentation testing plugin from ext-js [#621](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/621) ([#669](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/669)) ([a5170c4](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/a5170c494706a2bec3ba51e59966d0ca8a41d00e))

### Bug Fixes

* **aws-lambda:** BasicTracerProvider not force flushing ([#661](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/661)) ([76e0d0f](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/76e0d0fbef59e84c42b52d37cb3541e0dc853eb6))
* Update aws-lambda-instrumentation to SDK v0.25.0 ([#660](https://www.github.com/open-telemetry/opentelemetry-js-contrib/issues/660)) ([7b0d090](https://www.github.com/open-telemetry/opentelemetry-js-contrib/commit/7b0d0900bfb8475a32d799add4d925d7addbb24d))
</details>

This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

**Comment by dyladan** [comment]
Definitely a weird changelog but it seems to have worked

**Review by vmarchaud — APPROVED** [review]

File: packages/opentelemetry-test-utils/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opentelemetry/contrib-test-utils",
-  "version": "0.25.0",
+  "version": "0.26.0",
```
Comment (Flarna) [inline packages/opentelemetry-test-utils/package.json:3]: Is it intended to move a single package to a new version?

Comment (dyladan) [reply packages/opentelemetry-test-utils/package.json:3]: Yes. The automation is intended to allow individual package owners to make and release changes without waiting for other packages.

**Comment by dyladan** [comment]
Tests are failing because a package depends on unreleased test-util functionality.

File: packages/opentelemetry-test-utils/package.json (hunk 1 above)
Comment (Flarna) [reply packages/opentelemetry-test-utils/package.json:3]: ok.
As a result it's most likely better to remove contrib from the [compatibility matrix](https://github.com/open-telemetry/opentelemetry-js#compatibility-matrix) as there is no contrib version anymore.

**Review by Flarna — APPROVED** [review]

File: packages/opentelemetry-test-utils/package.json (hunk 1 above)
Comment (dyladan) [reply packages/opentelemetry-test-utils/package.json:3]: Yes I agree. We are going to have to put in each contrib package's README which SDK versions it is expected to work with.

**Comment by dyladan** [comment]
:robot: Release for @opentelemetry/contrib-test-utils is at https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/contrib-test-utils-v0.26.0 :sunflower:

**Comment by dyladan** [comment]
:robot: Release for @opentelemetry/instrumentation-aws-lambda is at https://github.com/open-telemetry/opentelemetry-js-contrib/releases/tag/instrumentation-aws-lambda-v0.26.0 :sunflower:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-mongodb :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/contrib-test-utils :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-aws-lambda :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-mongodb :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/contrib-test-utils :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-aws-lambda :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-mongodb :cloud:

**Comment by Flarna** [comment]
@dyladan  How often is the release createtion retried?

**Comment by dyladan** [comment]
I have no idea what happened here. It is not supposed to be retried at all

**Comment by dyladan** [comment]
I'm looking into it

**Comment by dyladan** [comment]
My guess is that it is because the branch wasn't deleted. I deleted it and we'll see if that happens again.

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/contrib-test-utils :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-aws-lambda :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-mongodb :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-aws-lambda :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-mongodb :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-aws-lambda :cloud:

**Comment by obecny** [comment]
@dyladan ^^

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-aws-lambda :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-aws-lambda :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-aws-lambda :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-mongodb :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-mongodb :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-mongodb :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/contrib-test-utils :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-aws-lambda :cloud:

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-aws-lambda :cloud:

**Comment by dyladan** [comment]
I'm asking in the release please repo why these messages keep happening

**Comment by dyladan** [comment]
:robot: Failed to create release for @opentelemetry/instrumentation-aws-lambda :cloud:

**Pull request merged**